packages/@cdktf/provider-generator/lib/get/generator/emitter/attributes-emitter.ts

```diff
@@ -167,6 +167,25 @@ export class AttributesEmitter {
     if (type.isBoolean) {
       return `this.getBooleanAttribute('${att.terraformName}')`;
     }
+
+    // TODO: enable this change once we want to release the next major version
```

Comment:
A (linked) follow-up issue for 0.16 would be nice, so we don't forget about this.